src/ddspdrum/torchmodule.py

```diff
@@ -108,76 +85,63 @@ def npyforward(
 
         return self.forward(*npyargs, **npykwargs).numpy()
 
-    # The following is cheesy AF but needed because the
-    # torchparameter is the master variable.
-    # TODO: Remove .value from modparameter
-    def _update_modparameters(self) -> None:
-        for modparameter_id in self.modparameters:
-            self.modparameters[modparameter_id].set_value(
-                float(self.torchparameters[modparameter_id].numpy())
-            )
-
-    # In general, we should consider removing all the following
-    # since we don't want to cast to int and then back to Tensor
-    # Alternately, we should make a SynthParameter which is an nn.Parameter
-    # but decorated with ModParameter stuff.
+    def add_parameters(self, parameters: List[TorchParameter]):
+        """
+        Add parameters to this SynthModule's torch parameter dictionary.
+        """
+        for parameter in parameters:
+            assert parameter.parameter_name not in self.torchparameters
+            self.torchparameters[parameter.parameter_name] = parameter
 
-    def get_modparameter(self, modparameter_id: str) -> ModParameter:
+    def get_parameter(self, parameter_id: str) -> TorchParameter:
         """
-        Get a single modparameter for this module
+        Get a single TorchParameter for this module
 
         Parameters
         ----------
-        modparameter_id (str)  :   Id of the modparameter to return
+        parameter_id (str)  :   Id of the parameter to return
         """
-        self._update_modparameters()
-        return self.modparameters[modparameter_id]
+        return self.torchparameters[parameter_id]
 
-    def get_modparameter_0to1(self, modparameter_id: str) -> float:
+    def get_parameter_0to1(self, parameter_id: str) -> float:
         """
-        Get the value of a single modparameter in the range of [0,1]
+        Get the value of a single parameter in the range of [0,1]
 
         Parameters
         ----------
         modparameter_id (str)  :   Id of the modparameter to return the value for
         """
-        self._update_modparameters()
-        return self.modparameters[modparameter_id].get_value_0to1()
+        return float(self.torchparameters[parameter_id].item())
 
-    def set_modparameter(self, modparameter_id: str, value: float):
+    def set_parameter(self, parameter_id: str, value: float):
```

Comment:
Maybe we should establish the convention that all these methods are either `_0to1` or `_human`/`_orig`??

Reply:
Don't have to tho

Reply:
Going to leave this for now, I think it makes sense at this level that an end-user would be interacting with it that you would want to set a parameter in the human range by default, and then having the other setter explicitly marked set_parameter_0to1. Happy to change this in the future if you find it confusing.